gui/__tests__/containers/SignIn.test.js

```diff
@@ -47,17 +47,24 @@ describe('containers/SignIn', () => {
 
   it('unauthenticated user inputs wrong credentials', async () => {
     const store = createStore()
-    global.fetch.postOnce(`/sessions`, { authenticated: false, errors: ['Invalid email'] })
+    global.fetch.postOnce(
+      '/sessions',
+      {authenticated: false, errors: ['Invalid email']},
+      {response: {status: 401}}
+    )
 
     const wrapper = mountSignIn(store)
     submitForm(wrapper)
 
     await syncFetch(wrapper)
 
-    expect(wrapper.text()).toContain('Invalid email')
     const newState = store.getState()
+    expect(newState.errors).toEqual({
+      messages: ['Your email or password are incorrect. Please try again'],
+      currentUrl: '/signin'
+    })
     expect(newState.session.authenticated).toEqual(false)
-    expect(newState.session.errors).toEqual(['Invalid email'])
+    expect(newState.session.errors).toEqual([])
```

Comment:
Let's add a test for `session.errors` in the event that it's not 401 so we have coverage. Otherwise, is it even being used? I'm inclined to remove `errors` on other reducers since they're seemingly unused and go solely with the new errors reducer. Let me know if I'm missing something.

Reply:
@dimroc I'm working on refactoring other errors so that they all get shown via the flash. Currently it's a hodge podge and some errors are sent as redux actions but not shown.